bambi/models.py

```diff
@@ -864,9 +864,16 @@ def graph(self, formatting="plain", name=None, figsize=None, dpi=300, fmt="png")
         fmt : str
             Format of the figure to save.
             Defaults to ``'png'``. Only works if ``name`` is not None.
+
+        Example
+        --------
+        > model = Model('y ~ x + (1|z)')
+        > model.build()
+        > model.graph()
+
         """
         if self.backend is None:
-            raise ValueError("The model is empty, please define a Bambi model")
```

Comment:
When I encountered this error, I thought it was strange because I was literally calling `model.graph()` - how could my model be not defined? I hope the new error message suggests the solution.

Reply:
Yes, you're completely right. Thanks for providing a better message!